repository/SmalltalkCI-Core.package/SmalltalkCI.class/class/failAndQuit..st

```diff
@@ -2,6 +2,8 @@ helpers
 failAndQuit: aString
 	"Prints and persists an internal failure for debugging purposes"
 	| message |
+	<haltOrBreakpointForTesting>
```

Comment:
ah, yes... this pragma will make the test to skip this method (otherwise it will report there is remaining "debug code" there.